configs/gtp-base.cfg

```diff
@@ -1,5 +1,3 @@
-@include compute/1gpu.cfg
```

Comment:
what's the reason for this change? was including `compute/1gpu.cfg` messing up Sabaki?

Reply:
Yeah the sabaki configs are designed to work with cpus only.